tests/regression/py_bindings/driver.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+
+import numpy
```

Comment:
Not important at all, but in Python the usual convention is to alias the numpy package as np (`import numpy as np`) and then call all its functions as `np.whatever()`

Reply:
done